---
title: 7.3 Histone Modifications in Cancer
---


## 7.3.1 Basic Mechanisms of Histone Modifications

Negatively charged DNA is packaged around a positively charged histone protein octamer, which contains 2 copies of histone proteins H2A, H2B, H3, and H4. This nucleoprotein complex is a nucleosome, the basic unit of chromatin. The nucleosomes of a continuous DNA polymer are connected by linker DNA and the complex is stabilized by histone protein H1.

N-terminal tails of core histone protein (H2A, H2B, H3, and H4) are largely targeted for the posttranscriptional modifications (PTMs), such as methylation, acetylation, and phosphorylation. Modifications in histone tails affect the chromatin structure which is critical for the gene regulation.

## 7.3.2 Types of Histone Modifications

The histone modifications include:

- **Arginine and lysine acetylation and methylation**
- **Serine phosphorylation**
- **Lysine ubiquitination**

All these changes are performed by enzymes such as:
- Histone acetyl-transferases (HATs)
- Histone methyl-transferases (HMTs)
- Histone deacetylases (HDACs)
- Histone demethylases (HDMs)

Histone deacetylases (HDACs) and histone demethylases (HDMs, or KDMs for lysine demethylases) remove acetyl- and methyl-groups from lysines, respectively. In addition, deubiquitinating enzymes contribute to the turnover of ubiquitin moieties.

## 7.3.3 Histone Modifications in Cancer

Chromatin structure is highly dynamic, and orchestrated by chromatin remodeling complexes, and histone modifying enzymes. Aberrations in histone modification caused by defects in activity of histone modifying enzymes and chromatin remodeling complexes may contribute to the neoplastic transformation.

Mutations in histone genes or chromatin modifier proteins are frequently detected in many cancer types, resulting in impairments in gene expression programs and genomic integrity. During the tumorigenesis, DNA methylation in promoter regions is induced, while genome-wide DNA methylation is downregulated.

**Clinical Examples in Bladder Cancer**

Chromatin deregulation is an emerging theme in cancer pathogenesis, and bladder cancer stands out among many other cancer types with frequent mutations of genes involved in epigenetic regulation. The most common mutations in bladder cancer occur in genes functioning in histone modification and chromatin remodeling genes. These include ARID1A (25%), KDM6A (24%), KMT2D (27%), EP300 (15%). Globally, almost 80% of all bladder cancer patients have mutations in genes involved in epigenetic regulation, demonstrating the high degree of epigenetic dysregulation in this cancer.

## 7.3.4 Metabolic Regulation of Histone Modifications

Most chromatin-modifying enzymes use metabolites as cofactors. Consequently, the cellular metabolism can influence the capacity of the cell to write or erase chromatin marks. This points to an intimate relationship between metabolic and epigenetic regulation.

S-adenosylmethionine (SAM) is the cofactor of DNA and histone methyltransferases and is the second most common enzymatic cofactor after ATP. SAM plays major roles in epigenetics, biosynthetic processes including phosphatidylcholine, creatine, and polyamine synthesis, as well as sulfur metabolism.
